src/style-value.js

```diff
@@ -29,21 +28,44 @@
       throw new TypeError('Can\'t parse an unsupported property.');
     }
 
-    // Make sure that there is no leading or trailing whitespace, case insensitive.
-    cssString = cssString.trim().toLowerCase();
-    if (propertyDictionary().isValidKeyword(property, cssString)) {
-      return new KeywordValue(cssString);
-    }
+    //Currently only supports sequences separated by ', '
+    var valueArray = cssString.split(', ');
+    var styleValueArray = [];
+    var supportedStyleValues =
+        propertyDictionary().getValidStyleValuesArray(property);
 
     var styleValueObject = null;
-    var supportedStyleValues = propertyDictionary().getValidStyleValuesArray(property);
-    for (var i = 0; i < supportedStyleValues.length; i++) {
-      styleValueObject = supportedStyleValues[i].parse(cssString);
-      if (styleValueObject != null) {
-        return styleValueObject;
+    var successfulParse = false;
+    for (var i = 0; i < valueArray.length; i++) {
+      var cssStringStyleValue = valueArray[i];
+      cssStringStyleValue = cssStringStyleValue.trim().toLowerCase();
```

Comment:
I think it'd be better to do the toLowerCase() on the whole cssString before you split it. Then you only need to trim here.